app/components/post_body_additional_content/index.js

```diff
@@ -52,11 +52,13 @@ function makeMapStateToProps() {
         const previewsEnabled = getBool(state, Preferences.CATEGORY_ADVANCED_SETTINGS, `${ViewTypes.FEATURE_TOGGLE_PREFIX}${ViewTypes.EMBED_PREVIEW}`) ||
             getBool(state, Preferences.CATEGORY_DISPLAY_SETTINGS, Preferences.LINK_PREVIEW_DISPLAY, true);
 
+        const openGraphData = getOpenGraphMetadataForUrl(state, link) || getOpenGraphData(ownProps.metadata, link);
+
         return {
             ...getDimensions(state),
             googleDeveloperKey: config.GoogleDeveloperKey,
             link,
-            openGraphData: getOpenGraphMetadataForUrl(state, link) || getOpenGraphData(ownProps.metadata, link)?.data,
+            openGraphData: openGraphData?.data,
```

Comment:
This is wrong.
`data` Property is only part of the second option in the or statement.